dagmc_stats/DagmcStats.py

```diff
@@ -54,8 +56,8 @@ def __set_native_ranges(self, entity_types):
                         Range of handles to that type
         """
 
-        self.native_ranges = {}
+        native_ranges = {}
         for entity_type in entity_types:
-            self.native_ranges[entity_type] = self._my_moab_core.get_entities_by_type(
+            native_ranges[entity_type] = self._my_moab_core.get_entities_by_type(
                 self.root_set, entity_type)
-        return self.native_ranges
+        return native_ranges
```

Comment:
No need to return anything - this is now a "set" method rather than a "get" method